### PR TITLE
fixed bug on number format for Dutch language.

### PR DESCRIPTION
**Pull request by chetanmaharishi**

### CODE REVIEW

File: grails-app/i18n/messages_nl.properties [hunk 1]
```diff
@@ -482,7 +482,7 @@ default.notes.label=Notities
 default.null.message=Eigenschap [{0}] van klasse [{1}] kan niet null zijn
 #default.number.format                 = 0
 default.numItems.label=\# items
-default.number.format=\#\#\#.\#\#\#.\#\#0,00
+default.number.format=###,###,##0.00
```
Comment (awalkowiak) [inline grails-app/i18n/messages_nl.properties:485]: I think all the `#` require backslash escape, because this file supports comments starting with `#`

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/i18n/messages_nl.properties (hunk 1 above)
Comment (jmiranda) [reply grails-app/i18n/messages_nl.properties:485]: I would have guessed you were right, but check messages.properties and all of the other .properties files.

Comment (awalkowiak) [reply grails-app/i18n/messages_nl.properties:485]: :thinking:

Comment (awalkowiak) [reply grails-app/i18n/messages_nl.properties:485]: Yup, looks like you're right, then you can disregard this comment, and it lgtm

**Pull request merged**